src/sec_limit_def.mbx

```diff
@@ -376,7 +374,7 @@
           <statement>
             <p>Which is given first in establishing a limit, the <m>x</m>-tolerance or the <m>y</m>-tolerance?</p>
 
-            <p><var name="$limitequalsvalue" form="popup"/> is given first.</p>
+            <p><var name="$limitequalsvalue" form="popup"/>is given first.</p>
```

Comment:
This one might matter. I'm just noting here. Might merge the PR and correct this one.